crates/extrinsics/src/lib.rs

```diff
@@ -363,6 +366,23 @@ impl ContractInfo {
     }
 }
 
+/// Fetch the contract wasm code from the storage using the provided client and code hash.
+pub async fn fetch_wasm_code(
+    hash: CodeHash,
+    client: &Client,
+) -> Result<Option<BoundedVec<u8>>> {
```

Comment:
Should convert the `BoundedVec` to a `Vec` for the return type: better not to expose types generated by subxt in the public api of this crate.

Reply:
Done